mlir/test/Dialect/Vector/vector-transfer-permutation-lowering.mlir

```diff
@@ -41,6 +41,25 @@ func.func @permutation_with_mask_scalable(%2: memref<?x?xf32>, %dim_1: index, %d
   return %1 : vector<8x[4]x2xf32>
 }
 
+// CHECK:           func.func @permutation_with_mask_transfer_write_scalable(
```

Comment:
Could you name the values here? (like the other tests)

Reply:
Sorry which values?